test/helpers.tsx

```diff
@@ -1,24 +1,118 @@
+import "date-fns";
 import { bigtestGlobals } from "@bigtest/globals";
 import { render as rtlRender } from "@testing-library/react";
-import { ReactElement } from "react";
+import { ComponentProps, ComponentType, ReactElement, useState } from "react";
 import { StylesProvider, jssPreset } from "@material-ui/core/styles";
 import { create } from "jss";
+import { MuiPickersUtilsProvider } from "@material-ui/pickers";
+import DateFnsUtils from "@date-io/date-fns";
 
-export function render(element: ReactElement) {
-  let insertion = bigtestGlobals.document.createComment("mui-jss-insertion");
-  let insertionPoint = bigtestGlobals.document.head.insertBefore(insertion, bigtestGlobals.document.head.firstChild);
-  const jss = create({
-    ...jssPreset(),
-    // Define a custom insertion point that JSS will look for when injecting the styles into the DOM.
-    insertionPoint,
-  });
-
-  return rtlRender(
-    <StylesProvider jss={jss} injectFirst>
-      {element}
-    </StylesProvider>,
-    {
-      container: bigtestGlobals.document.body,
-    }
-  );
+interface StepImplementation {
+  description: string;
+  action: () => void;
+}
```

Comment:
It just occured to me that we should be using this interface. https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/interaction.ts#L12-L21

let's not do it in this PR though.

Reply:
It can't be used so easily, we need to define `then, catch` and other methods, because Interaction extends from Promise. But step method doesn't have mandatory of these methods

Reply:
Yeah, let's revisit this later, because this is perfect for now. What we really need is to have a way to import this interface https://github.com/thefrontside/bigtest/blob/v0/packages/suite/src/interfaces.ts#L45-L52 directly from `@bigtest/interactor` so that material-ui-interactors only has one dependency and doesn't need to depend on `@bigtest/suite`